custom_dc/custom.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2023 Google LLC
```

Comment:
should we put this file in app_env? Curious why the rest of the environment config files are there but this one is separate?

Reply:
nit: should we rename this file to custom_env.py for more clarity?